src/wazuh_modules/vulnerability_detector/cpe_helper.json

```diff
@@ -36,11 +89,11 @@
                     "^Sun"
                 ],
                 "product": [
-                    "Java .* Development Kit",
-                    "^Oracle.*VirtualBox [0-9]",
+                    "Java.* Development Kit",
+                    "Oracle.*VirtualBox.*",
```

Comment:
- Is there any case where it does not start with `Oracle` so that the '`^`' regex has to be removed?
- Does the version no longer appear after the `VirtualBox` product name? Could you give me an example of the case where it has to match, please?